ep/build-clickhouse/src/build_clickhouse.sh

```diff
@@ -16,5 +23,5 @@ done
 
 echo ${GLUTEN_SOURCE}
 
-cmake -G Ninja -S ${GLUTEN_SOURCE}/cpp-ch -B ${GLUTEN_SOURCE}/cpp-ch/build_ch -DCH_SOURCE_DIR=${GLUTEN_SOURCE}/cpp-ch/ClickHouse "-DCMAKE_C_COMPILER=$(command -v clang-15)" "-DCMAKE_CXX_COMPILER=$(command -v clang++-15)" "-DCMAKE_BUILD_TYPE=Release"
+cmake -G Ninja -S ${GLUTEN_SOURCE}/cpp-ch -B ${GLUTEN_SOURCE}/cpp-ch/build_ch -DCH_SOURCE_DIR=${CH_SOURCE_DIR} "-DCMAKE_C_COMPILER=$(command -v clang-15)" "-DCMAKE_CXX_COMPILER=$(command -v clang++-15)" "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
```

Comment:
`RelWithDebInfo` is fine, but `libc.so` is too large, i am not sure how it affects CI.

@liuneng1994  any idea?

Reply:
ci doesn't use the script

Reply:
OK